apps/prairielearn/assets/scripts/instructorAssessmentManualGradingInstanceQuestion.js

```diff
@@ -206,19 +218,26 @@ function updateSettingsPointValues() {
 
 function checkRubricItemTotals() {
   const form = document.querySelector('.js-rubric-settings-modal form');
+  if (!form) {
+    return;
+  }
   const startingPoints = Number(form.querySelector('[name="starting_points"]:checked')?.value ?? 0);
   const [totalPositive, totalNegative] = Array.from(form.querySelectorAll('.js-rubric-item-points'))
     .map((input) => Number(input.value))
     .reduce(
       ([pos, neg], value) => (value > 0 ? [pos + value, neg] : [pos, neg + value]),
       [startingPoints, startingPoints],
     );
-  const minPoints = Number(form.querySelector('[name="min_points"]').value);
+  const minPoints = Number(form.querySelector('[name="min_points"]')?.value ?? 0);
```

Comment:
Another one of these; I'm a little torn on what to do here. I'd be somewhat more inclined to try to fail loudly if what seems to be a necessary input isn't present?

I suppose this will all be rewritten in strongly-typed Preact soon and this problem will go away 🙂 I do think this is another concrete benefit of using Preact: Zod lets us ensure that our starting state is valid and strongly-typed _and_ eliminates pretty much the entire "is this DOM node/attribute/etc. present" class of problem.

Reply:
Added these three to fail loudly; but as this code will be migrated soon I won't do the rest.